zerodark-theme.el

```diff
@@ -63,6 +63,10 @@
   '((t :foreground "#98be65" :height 0.9))
   "Face for not modified buffers in the mode-line.")
 
+(defface zerodark-buffer-position-alt-face
+  '((t :foreground "#cccccc" :height 0.9))
```

Comment:
`#cccccc` is the same color as the active-mode-line, isn't it? If that's what you want, then we can just remove the face property.

If it's not what you want, you could choose a slightly darker grey to dim it a bit.